### DISCUSSION TITLE
Show Multiple nodes on dragging one node

### DISCUSSION THREAD

**Discussion by Hamza-malikx** [xyflow/xyflow#2239] [Q&A]
Hi, react-flow team.
I am trying to implement a functionality where if I drag one node from sidebar, I wanted to show the user three nodes along with edges on canvas. For one time, I have achieved this, like a user can drag one node, based on node value I am setting up (a sort of initial nodes and edges) which shows up but when I do the same procedure again it instantiates the state. 
I hope u guys got my point.  
![ezgif com-gif-maker](https://user-images.githubusercontent.com/83468742/175012866-3b7a5c28-ded1-490c-bc4a-1a85e8a9dc35.gif)

**Comment by moklick** [comment]
I see the problem but without any code (repo or condesandbox) we can't help you here.

**Reply by Hamza-malikx** [reply]
> I see the problem but without any code (repo or condesandbox) we can't help you here.

Ah thank you, I have fixed the issue.
I was actually passing the static ids to each node (instead of generating new random ids) whenever I was dropping a new node.
